dependencymagnet/doc.go

```diff
@@ -1,7 +1,9 @@
 package dependencymagnet
 
 import (
-	// alpha-build-machinery has non-go code needed for makefiles.
-	// Needed to work with go modules.
-	_ "github.com/openshift/library-go/alpha-build-machinery"
+	_ "github.com/openshift/build-machinery-go/make"
```

Comment:
fyi I prefer this more precise, but multiple line, form but @adambkaplan called it out in my k8s rebase OCM PR, 
preferring just a single `"github.com/openshift/build-machinery-go"` entry

I'm happy with this as is but if you want to consolidate I won't object

Reply:
as Gabe alluded, I prefer just a single import from `github.com/openshift/build-machinery-go`. This project is a bag of makefiles and scripts, so golang has no means of deciphering dependencies.

Reply:
Updated, PTAL.